test/drop_test/config.py

```diff
@@ -137,6 +137,12 @@ def __init__(self, size: int, id: str):
     "utf8-testfile-\u5b81\u5BDF": TestFile(
         size=1 * 1024, id="iYHFSaSzwOXygOqayOaVNlF4cQUF2nyZyTzaIpnVau8"
     ),
+    "</file-01": TestFile(
+        size=1 * 1024, id="vb81tDWO3JDf_CG-MFw_pMMgOLTMxJIhmjBAVcglshk"
+    ),
+    ">/file-01": TestFile(
+        size=1 * 1024, id="d0Gycotel31QlJnnMP7hk_JKSyImdt8Fwsvp8uts2_8"
+    ),
```

Comment:
I tripped on these paths because they look so unnatural, what do you think about doing something along the lines `</file-01` -> `contains-invalid-<-characters/file-01`? This way it would be in the middle and a bit more obvious

Reply:
Done